confuse.py

```diff
@@ -1552,7 +1593,7 @@ def as_template(value):
             and issubclass(value, enum.Enum)):
         return Choice(value)
     elif isinstance(value, list):
-        return OneOf(value)
+        return SequenceTemplate(value)
```

Comment:
You're right that changing this would break some existing code that uses `OneOf` templates defined by lists. In general, it's an imperfect process translating from the little "DSL" of Python objects into proper `Template`s, so it's not too distressing that there's a conflict here—both of these could be naturally expressed as `[something]`.

For now, I think we should just leave this branch as it is. It's not too hard to import `Sequence` and use `.get(Sequence({}))` instead of `.get([{}])`, for example. This reminds me of `List[float]` in Python 3's standard type annotations, for example. We could even provide a shorthand name like `seq` to make this more compact to type.